prowler/providers/gcp/gcp_provider.py

```diff
@@ -770,6 +788,28 @@ def validate_static_arguments(
             "type": "authorized_user",
         }
 
+    def is_project_matching(self, input_project: str, project_to_match: str) -> bool:
```

Comment:
Can you restore this to their original location?

Reply:
Done